app/models/comment.rb

```diff
@@ -1,4 +1,6 @@
 class Comment < ApplicationRecord
-  belongs_to :author_id
-  belongs_to :post_id
+  belongs_to :user
+  belongs_to :post
+
+  scope :last_post_comments, ->(post_id) { where(post_id:).last(5) }
```

Comment:
- [ ] I think this method it should be in post model since you want to display the last 5 recent comments for a post so it would be better if you did it in post model 🔥💯🙏🏻